G1ANT.Addon.Images/G1ANT.Addon.Images.Tests/Commands/image.expected/ShouldFindImage_WhenImageIsExpected.G1ANT

```diff
@@ -0,0 +1,11 @@
+﻿addon images version 2.205.17304.1000
+
+♥guid = ⊂Guid.NewGuid().ToString()⊃
+♥imageFile = ♥environment⟦temp⟧\♥guid.jpg
+file.download https://jeremykun.files.wordpress.com/2012/01/img49.png filename ♥imageFile
+program mspaint.exe arguments ♥imageFile
```

Comment:
You don't close mspaint.exe anywhere. It causes a problem when running other tests

Reply:
keyboard window ♥activeWindow text ⋘ALT+F4⋙

Reply:
sorry, something didnt work on my machine :D it's ok